src/main/java/roomescape/service/TimeService.java

```diff
@@ -0,0 +1,47 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.dto.TimeRequestDto;
+import roomescape.dto.TimeResponseDto;
+import roomescape.entity.Time;
+import roomescape.repository.TimeRepository;
+
+import java.time.LocalTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TimeService {
+
+    private final TimeRepository timeRespository;
+
+    public TimeService(TimeRepository timeRespository) {
+        this.timeRespository = timeRespository;
+    }
+
+    public List<TimeResponseDto> findAllTimes() {
+        return timeRespository.findAll()
+                .stream()
+                .map(time -> new TimeResponseDto(
+                        time.getId(),
+                        time.getTimeASALocalTime()))
+                .collect(Collectors.toList());
```

Comment:
아래처럼 변경할 수 있어요.
```suggestion
                .toList();
```

약간의 차이점이 있으니 알고계시면 좋습니다!